images/win/scripts/Installers/Install-DotnetSDK.ps1

```diff
@@ -49,22 +52,24 @@ function InstallSDKVersion (
 
 function InstallAllValidSdks()
 {
-    Invoke-WebRequest -Uri 'https://raw.githubusercontent.com/dotnet/core/master/release-notes/releases-index.json' -UseBasicParsing -OutFile 'releases-index.json'
-    $dotnetChannels = Get-Content -Path 'releases-index.json' | ConvertFrom-Json
+    $releasesIndexPath = Start-DownloadWithRetry -Url 'https://raw.githubusercontent.com/dotnet/core/master/release-notes/releases-index.json' -Name 'releases-index.json'
+    $dotnetChannels = Get-Content -Path $releasesIndexPath | ConvertFrom-Json
```

Comment:
I am a bit confused by `$releasesIndexPath` variable. Looks like you are replacing `releases-index.json` with it, but at the same time on the line above you are passing it in raw string. 
Could it be refactored to something like this?
```
$releasesIndexPath = 'releases-index.json'
$releaseUrl = 'https://raw.githubusercontent.com/dotnet/core/master/release-notes/releases-index.json'
Start-DownloadWithRetry -Url $releaseUrl -Name $releasesIndexPath
$dotnetChannels = Get-Content -Path $releasesIndexPath | ConvertFrom-Json
```

Reply:
Fixed. Thanks!